package.json

```
@@ -19,7 +19,9 @@
   ],
   "scripts": {
     "build": "tsc --build ./tsconfig.build.json",
+    "build:clean": "yarn clean && yarn build",
     "build:docs": "retype build",
+    "clean": "yarn build --clean",
```

Doesn't this clean _and_ build? I was thinking something more like:
```suggestion
    "clean": "rimraf dist '**/*.tsbuildinfo'",
```

---

It doesn't. `tsc --build --clean` just cleans the dist folders.